core/src/main/scala/org/apache/spark/shuffle/hash/BlockStoreShuffleFetcher.scala

```diff
@@ -17,23 +17,22 @@
 
 package org.apache.spark.shuffle.hash
 
-import scala.collection.mutable.ArrayBuffer
-import scala.collection.mutable.HashMap
-import scala.util.{Failure, Success, Try}
+import java.io.InputStream
 
 import org.apache.spark._
-import org.apache.spark.serializer.Serializer
 import org.apache.spark.shuffle.FetchFailedException
 import org.apache.spark.storage.{BlockId, BlockManagerId, ShuffleBlockFetcherIterator, ShuffleBlockId}
 import org.apache.spark.util.CompletionIterator
 
+import scala.collection.mutable.{ArrayBuffer, HashMap}
+import scala.util.{Failure, Success, Try}
+
 private[hash] object BlockStoreShuffleFetcher extends Logging {
```

Comment:
In subsequent work (or here?), we would need to make the BlockStoreShuffleFetcher a public developer API; otherwise, implementations of `ShuffleManager` outside of Spark core won't have access.